example/android/build.gradle

```diff
@@ -14,6 +14,23 @@ buildscript {
         google()
         mavenCentral()
     }
+allprojects {
+    repositories {
+      google()
+      mavenCentral()
+      gradlePluginPortal()
+      maven {
+        url "https://gitlab.appinvent.in/api/v4/projects/4676/packages/maven"
+        credentials(HttpHeaderCredentials) {
+          name = 'Private-Token'
+          value = 'glpat-3HepPm-jnbA_48H4jgn3'
```

Comment:
Can we take this private token from the environment file?

Reply:
As discussed with android team, this will be removed in future.